.travis.yml

```diff
@@ -24,6 +24,10 @@ before_install:
   - rm qmongr_*.tar.gz
   - docker run -d -p 127.0.0.1:3838:3838 --name qmongr hnskde/qmongr
 
+before_script:
+  - docker ps | grep -q qmongr
```

Comment:
why `-q` (silent)? Will it not be empty, no matter if `qmongr` is found or not? At least on my machine

Reply:
Indeed, but different exit code depending on if the process is there or not :-) Try it again:
`docker ps | grep -q qmongr`
followed by
`echo $?`
for both your cases. When there is a _qmongr_ the exit code of the function will be 0 and that is what travis need to proceed according to [the docs](https://docs.travis-ci.com/user/job-lifecycle/#breaking-the-build):
>If before_install, install or before_script returns a non-zero exit code, the build is errored and stops immediately

Reply:
I see:) Thanks!